src/connectedk8s/azext_connectedk8s/tests/latest/test_connectedk8s_scenario.py

```diff
@@ -7,7 +7,7 @@
 import unittest
 
 from azure.cli.testsdk import (LiveScenarioTest, ResourceGroupPreparer)  # pylint: disable=import-error
-from azure.cli.testsdk.scenario_tests import AllowLargeResponse  # pylint: disable=import-error
+#from azure.cli.testsdk.scenario_tests import AllowLargeResponse  # pylint: disable=import-error
```

Comment:
> #from azure.cli.testsdk.scenario_tests import AllowLargeResponse  # pylint: disable=import-error

Why not remove duplicate imports directly?

Reply:
removed the unused import